src/main/java/org/spongepowered/common/util/gen/ArrayMutableBlockBuffer.java

```diff
@@ -66,16 +70,10 @@ public ArrayMutableBlockBuffer(BlockPalette palette, Vector3i start, Vector3i si
         this.data = new CharBackingData(blocks);
     }
 
-    public ArrayMutableBlockBuffer(BlockPalette palette, Vector3i start, Vector3i size, int[] blocks) {
-        super(start, size);
-        this.palette = palette;
-        this.data = new IntBackingData(blocks);
-    }
-
     ArrayMutableBlockBuffer(BlockPalette palette, Vector3i start, Vector3i size, BackingData blocks) {
         super(start, size);
         this.palette = palette;
-        this.data = blocks.copyOf();
```

Comment:
Or we could remove the BlockPalette's remove() method... I don't like that method anyway, supporting it is why BimapPalette has a BitSet. (nothing in SpongeForge or any of its subprojects use it as far as I know, so its only there for the plugins)